src/main/java/teammates/ui/template/FeedbackResponseRow.java

```diff
@@ -98,13 +88,17 @@ public String getResponseText() {
      */
     public String getTypeOfPeopleCanViewComment(FeedbackResponseCommentAttributes comment,
                                                 FeedbackQuestionAttributes relatedQuestion) {
```

Comment:
This function is duplicated in `PageData` but it can be used it here because of macker rule.

Reply:
Or put it in constructor? For that we can pass `FeedbackQuestionAttributes question` through constructor and then use `getTypeOfPeopleCanViewComment`?

Reply:
Also can if it is compulsory to have this setting (I think yes as we now remove the checking).